server/aws-lsp-codewhisperer/src/language-server/netTransform/transformHandler.ts

```diff
@@ -0,0 +1,255 @@
+import path = require('path')
+import { Workspace } from '@aws/language-server-runtimes/out/features'
+import * as fs from 'fs'
+import * as os from 'os'
+import { v4 as uuidv4 } from 'uuid'
+import {
+    CreateUploadUrlResponse,
+    GetTransformationRequest,
+    StopTransformationRequest,
+} from '../../client/token/codewhispererbearertokenclient'
+import { CodeWhispererServiceToken } from '../codeWhispererService'
+import { getCWStartTransformRequest, getCWStartTransformResponse } from './converter'
+import {
+    CancellationJobStatus,
+    QNetCancelTransformRequest,
+    QNetCancelTransformResponse,
+    QNetGetTransformPlanRequest,
+    QNetGetTransformPlanResponse,
+    QNetGetTransformRequest,
+    QNetGetTransformResponse,
+    QNetStartTransformRequest,
+    QNetStartTransformResponse,
+} from './models'
+import { cleanup, createZip, getSha256 } from './utils'
+import fetch = require('node-fetch')
+
+export class TransformHandler {
+    private client: CodeWhispererServiceToken
+    private workspace: Workspace
+    constructor(client: CodeWhispererServiceToken, workspace: Workspace) {
+        this.client = client
+        this.workspace = workspace
+    }
+
+    async startTransformation(userInputrequest: QNetStartTransformRequest): Promise<QNetStartTransformResponse> {
+        try {
+            const uploadId = await this.preTransformationUploadCode(userInputrequest)
+            const request = getCWStartTransformRequest(userInputrequest, uploadId)
+            console.log('send request to start transform api: ' + JSON.stringify(request))
```

Comment:
I think this will work in most clients, but I'm not actually sure whether this is a client or a server concern. Looking at this (https://github.com/microsoft/vscode-languageserver-node/issues/402#issuecomment-432609554) it looks like it's the client.

When in doubt, we should probably use the `logging` feature: https://github.com/aws/language-server-runtimes/blob/main/src/features/logging.ts to ensure this output goes to the client and can be inspected in the output channel.

Reply:
Ack. Will replace all console log with `logging`